test/Multiformats.Hash.Tests/Multiformats.Hash.Tests.csproj

```diff
@@ -20,22 +20,30 @@
     <DefineConstants>$(DefineConstants)</DefineConstants>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <!-- https://github.com/tonerdo/coverlet -->
+    <!-- Do not collect by default -->
+    <CollectCoverage>false</CollectCoverage>
```

Comment:
Here are all the properties for codecov

Reply:
My bad. I was confused because one check from codecov fails. Looks good to me.